sdk/src/main/java/io/opentelemetry/sdk/trace/SpanWrapper.java

```diff
@@ -58,9 +64,21 @@ static SpanWrapper create(
       ReadableAttributes attributes,
       int totalAttributeCount,
       int totalRecordedEvents,
-      Status status) {
+      Status status,
+      String name,
+      long endEpochNanos,
+      boolean hasEnded) {
     return new AutoValue_SpanWrapper(
-        delegate, links, events, attributes, totalAttributeCount, totalRecordedEvents, status);
+        delegate,
```

Comment:
I think a small comment can help

`// We reduce overhead by using the delegate for all immutable fields. Take care when adding something new to this class when determining whether to delegate or not.`

Reply:
I'm hesistant to add this in this bugfix PR. It might cause lots of discussion on how to word it. That aspect you want me to call out did not really change with this PR.

Reply:
Well the main reason it came to mind is while reviewing this I had to make sure we didn't miss other fields. I think with this PR you put in the thought so documenting that is worthwhile. If this sort of small comment can cause a bikeshed we're in trouble :(

Reply:
OK, I will add it 😄

Reply:
Added a javadoc in cc79fdb.

Reply:
Thanks, I really like it!